strawberry/exceptions.py

```diff
@@ -246,3 +250,12 @@ def __init__(self):
             "that inherits from BaseContext or a dictionary"
         )
         super().__init__(message)
+
+
+class UncallableResolverError(Exception):
+    def __init__(self, resolver: "StrawberryResolver"):
```

Comment:
Quotes around `StrawberryResolver` are unnecessary because we're using `__future__` annotations